docs/_includes/person.html

```diff
@@ -0,0 +1,36 @@
+<h2>{{ include.title }}</h2>
+{% comment %}
+We want to sort people by surname.
```

Comment:
Would it be easier to have a "last_name" entry in the yaml and concatenate them here, or have a "sort_key" entry that doesn't even get displayed?

Reply:
I thought about this and I'm not sure which route is correct.  I see two options:

 - Add a sort key and also a unique identifier (github username, maybe?) that we can use to refer to people from blog posts.
 - Do the current set of hacks to sort things.

If we go with the second one, it works for now and there's some fairly mechanical transformation needed to get to the other approach.  I'm starting to think it is better to do that now, rather than have to update it later.

Reply:
I've added a sort key.  This simplifies everything and seems nicer.